tools/generators/pbxnativetargets/src/Generator/TargetArguments.swift

```diff
@@ -80,19 +80,22 @@ extension Dictionary<TargetID, TargetArguments> {
                 try rawArgs.consumeArg("has-c-params", as: Bool.self, in: url)
             let hasCxxParams =
                 try rawArgs.consumeArg("has-cxx-params", as: Bool.self, in: url)
-            let srcs =
-                try rawArgs.consumeArgs("srcs", as: BazelPath.self, in: url)
-            let nonArcSrcs = try rawArgs.consumeArgs(
+            let srcs = try rawArgs.consumeArgsUntilNull(
+                "srcs",
+                as: BazelPath.self,
+                in: url
+            )
+            let nonArcSrcs = try rawArgs.consumeArgsUntilNull(
                 "non-arc-srcs",
                 as: BazelPath.self,
                 in: url
             )
-            let resources = try rawArgs.consumeArgs(
+            let resources = try rawArgs.consumeArgsUntilNull(
```

Comment:
I'll try to change other uses of `consumeArgs` to `consumeArgsUntilNull` in a future change. It has lower Starlark CPU use. After that, it might be the only version of `consumeArgs` remaining, in which I'll rename `consumeArgsUntilNull` to `consumeArgs`.